pywb/templates/banner.html

```diff
@@ -0,0 +1,31 @@
+{% if not env.pywb_proxy_magic or config.proxy.enable_banner | default(true) %}
```

Comment:
Can you drop one or more comments to this template to make clear which bits of it are overridden/changed?